### ISSUE TITLE
[BUG] Errors on generation

### ISSUE THREAD

**Issue by rx-dev** [OrangeUtan/generator-beet#1]
```bash
@ yo beet:datapack EnderChest2
? Project name ec
? Description i
? Author rx
Error beet:datapack EnderChest2 

this.props.author.toLowerCase(...).replaceAll is not a function
```

Unsure how to move forward

**Comment by OrangeUtan** [comment]
`toLowerCase` and `replaceAll` are both String functions, so if the type or value of the author was the issue, it would have thrown an error earlier.

I suspect your nodejs version is older than the required version, thus `replaceAll` does not exist. Can you tell me your nodejs version?

I will verify this later and add a required nodejs version. Making the generator backwards compatible with your version would be a good idea as well, since the selected version is not an absolute requirement.

**Comment by OrangeUtan** [comment]
Yes, as suspected, Nodejs 14 LTS does not support `replaceAll` yet.
I have neglected this until now, but I think its time that I add tests for bot Node 14 & 16.
